core/backend/src/test/standalone/TxnManager.test.ts

```diff
@@ -15,7 +15,8 @@ import {
 } from "../../imodeljs-backend";
 import { IModelTestUtils, TestElementDrivesElement, TestPhysicalObject, TestPhysicalObjectProps } from "../IModelTestUtils";
 
-describe("TxnManager", () => {
+// TEMPORARILY disabled until we can figure out why they fail on CI jobs randomly

```

Comment:
I have only seen two of these tests fail in CI:
- "change propagation should leave txn empty" and
- "dispatches events when elements change".

Have you seen others fail?

Reply:
I have also seen "dispatches events in batches" fail. I don't intend for this to be disabled for long.